input/pagecontent/acteurs-transactions.md

```diff
@@ -0,0 +1,342 @@
+#### Liste des Acteurs et systèmes concernés
+
+Le présent volet met en œuvre les Acteurs IHE suivants, représentant le
+rôle joué par un ou plusieurs composants du système d'information :
```

Comment:
Il y a un saut de ligne en trop

Reply:
(cas dans tous les md) - ça ne pose pas de soucis d'affichage ?

Reply:
Cette page est vide : https://ansforge.github.io/IG-hl7v2-volet-transmission-document-cda-r2/sd-tranfo-pdf-to-ig/ig/role.html

Reply:
> Cette page est vide : https://ansforge.github.io/IG-hl7v2-volet-transmission-document-cda-r2/sd-tranfo-pdf-to-ig/ig/role.html

vu après :)
contenu ajouté

Reply:
> (cas dans tous les md) - ça ne pose pas de soucis d'affichage ?

Apparemment non